solution/0000-0099/0029.Divide Two Integers/README.md

```diff
@@ -227,6 +227,44 @@ public class Solution {
 }
 ```
 
+```php
+class Solution {
+    /**
+     * @param integer $dividend
+     * @param integer $divisor
+     * @return integer
+     */
+
+    function divide($dividend, $divisor) {
+        if ($divisor == 0) {
```

Comment:
Thanks for your contribution. You can refer to the naming in other languages to facilitate the reading experience of switching between multiple languages.

![image](https://github.com/doocs/leetcode/assets/117748716/67883861-7254-4bf3-b206-6282ee658630)

Reply:
Thank you! I will have a  look on it.